config.py

```diff
@@ -104,6 +104,7 @@
 
 ACCESS_CONTROLS = {'render test': {
     'allowrooms': ('coala/cobot-test', 'coala/corobo',)},
-    'LabHub:*': {'allowprivate': False}}
+    'LabHub:*': {'allowprivate': False}
```

Comment:
so this is a syntax error.
not tested.
and something is wrong with CI that it didnt find this problem for you.
please fix.  separate PR.